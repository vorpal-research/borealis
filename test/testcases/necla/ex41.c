extern int __NONDET__();
  
extern char x[101], y[101], z[201];
int from,to,i,j,k;

void mainFunc() {  
  from = __NONDET__();
  to = __NONDET__();
  k = __NONDET__();

  i = from;
  j = 0;
  while(x[i] != 0 && i < to){
    z[j] = x[i];
    i++;
    j++;
  }
  
  if(k >= 0 && k < j)
    if(z[k] == 0)
      {ERROR: goto ERROR;}  /* prove strlen(z) == j */
}
