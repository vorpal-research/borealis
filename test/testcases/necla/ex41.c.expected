
[
	
	{
		"location" : 
		{
			"filename" : "test/testcases/necla/ex41.c",
			"loc" : 
			{
				"col" : 3,
				"line" : 15
			}
		},
		"type" : "BUF-01"
	},
	
	{
		"location" : 
		{
			"filename" : "test/testcases/necla/ex41.c",
			"loc" : 
			{
				"col" : 5,
				"line" : 16
			}
		},
		"type" : "BUF-01"
	},
	
	{
		"location" : 
		{
			"filename" : "test/testcases/necla/ex41.c",
			"loc" : 
			{
				"col" : 8,
				"line" : 22
			}
		},
		"type" : "BUF-01"
	}
]
